spec/views/idv/address/new.html.erb_spec.rb

```diff
@@ -15,9 +23,16 @@
       if gpo_letter_requested
         expect(parsed_page).to have_content(t('doc_auth.headings.mailing_address'))
         expect(parsed_page).to have_content(t('doc_auth.info.mailing_address'))
+        expect(parsed_page).to have_content(t('forms.buttons.continue'))
```

Comment:
Do we want to test for `Back` vs `Cancel` in all cases?

Reply:
Yes updated.